gratipay/models/participant/routes.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import braintree
+from psycopg2 import IntegrityError, errorcodes
+
+from ..exchange_route import ExchangeRoute
+from ...billing.instruments import CreditCard
+from ...utils import is_card_expiring
+
+
+class Routes(object):
```

Comment:
Fair enough, done in 91a567cdcd69a614b9d26784aba665f5aac63c57. That's a nomenclature wart for sure.